.github/workflows/ci.yml

```diff
@@ -62,6 +62,7 @@ jobs:
 
   integration_tests:
     name: Integration testing on - ${{ matrix.os }}.
```

Comment:
This should be named cloud integration test? @galibey in that case why k3 is needed?

Reply:
@sehz these tests run locally. For cloud tests we usually create another workflow and run on intervals

Reply:
Then cloud login should be removed? This should be strictly run in locally